FastExcel/FastExcel.cs

```diff
@@ -89,6 +89,8 @@ internal void PrepareArchive(bool openSharedStrings = true)
             {
                 SharedStrings = new SharedStrings(Archive);
             }
+
+            loadDefinedNames();
```

Comment:
Should we add private variable, and in the getter for `FastExcel.DefinedNames` check that variable for null value and load then, otherwise return private variable? This would make it load on demand. How would you name said private variable if we were to go this way?